devfiles/quarkus-command-mode/devfile.yaml

```diff
@@ -0,0 +1,100 @@
+---
+apiVersion: 1.0.0
+metadata:
+  generateName: quarkus-commandmode-
+projects:
+  -
+    name: quarkus-quickstarts
+    source:
+      type: git
+      location: "https://github.com/quarkusio/quarkus-quickstarts.git"
+      sparseCheckoutDir: getting-started-command-mode
+components:
+  -
+    type: chePlugin
+    id: redhat/quarkus-java11/latest
+  -
+    type: dockerimage
+    alias: centos-quarkus-maven
+    image: quay.io/eclipse/che-quarkus:nightly
+    env:
+      - name: JAVA_OPTS
+        value: "-XX:MaxRAMPercentage=50.0 -XX:+UseParallelGC -XX:MinHeapFreeRatio=10
+          -XX:MaxHeapFreeRatio=20 -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90
+          -Dsun.zip.disableMemoryMapping=true -Xms20m -Djava.security.egd=file:/dev/./urandom
+          -Duser.home=/home/user"
+      - name: MAVEN_OPTS
+        value: $(JAVA_OPTS)  
+    memoryLimit: 4927M
+    mountSources: true
+    volumes:
+      - name: m2
+        containerPath: /home/user/.m2
+
+  - type: dockerimage
+    alias: ubi-minimal
+    image: 'registry.access.redhat.com/ubi8/ubi-minimal'
+    memoryLimit: 32M
+    mountSources: true
+    command: ['tail']
+    args: ['-f', '/dev/null']
+
+commands:
+  -
+    name: Package
+    actions:
+      -
+        type: exec
+        component: centos-quarkus-maven
+        command: "mvn package"
+        workdir: ${CHE_PROJECTS_ROOT}/quarkus-quickstarts/getting-started-command-mode
+  -
+    name: Start Development mode (Hot reload + debug)
+    actions:
+      -
+        type: exec
+        component: centos-quarkus-maven
+        command: >-
+          echo Enter the command arguments;
+          read arguments;
+          set -o xtrace;
+          mvn compile quarkus:dev -Dquarkus.args="$arguments"
+        workdir: ${CHE_PROJECTS_ROOT}/quarkus-quickstarts/getting-started-command-mode
+  -
+    name: Package Native
+    actions:
+      -
+        type: exec
+        component: centos-quarkus-maven
+        command: "mvn package -Dnative -Dmaven.test.skip"
+        workdir: ${CHE_PROJECTS_ROOT}/quarkus-quickstarts/getting-started-command-mode
+
+  -
+    name: Start Native
+    actions:
+      -
+        type: exec
+        component: ubi-minimal
+        command: >-
+          echo Enter the command arguments;
```

Comment:
could we add a semi-colon ?

when testing it was not obvious that something was expected ?

or like adding prompt ` > ?`